package.json

```diff
@@ -19,19 +19,19 @@
     "Gregor Martynus (https://twitter.com/gr2m)"
   ],
   "dependencies": {
-    "@semantic-release/error": "^4.0.0",
-    "aggregate-error": "^5.0.0",
-    "execa": "^8.0.0",
-    "fs-extra": "^11.0.0",
-    "lodash-es": "^4.17.21",
-    "nerf-dart": "^1.0.0",
-    "normalize-url": "^8.0.0",
-    "npm": "^10.5.0",
-    "rc": "^1.2.8",
-    "read-pkg": "^9.0.0",
-    "registry-auth-token": "^5.0.0",
-    "semver": "^7.1.2",
-    "tempy": "^3.0.0"
+    "@semantic-release/error": "4.0.0",
+    "aggregate-error": "5.0.0",
+    "execa": "8.0.1",
+    "fs-extra": "11.2.0",
+    "lodash-es": "4.17.21",
+    "nerf-dart": "1.0.0",
+    "normalize-url": "8.0.1",
+    "npm": "10.6.0",
+    "rc": "1.2.8",
+    "read-pkg": "9.0.1",
+    "registry-auth-token": "5.0.2",
+    "semver": "7.6.0",
+    "tempy": "3.1.0"
```

Comment:
we intentionally do not pin these versions in order to enable consumers to update to newer versions without needing to wait for a release of this package and also to enable npm to dedupe more effectively